tests/phpunit/tests/cron.php

```diff
@@ -1112,4 +1112,58 @@ function() {
 		$this->assertSame( 'could_not_set', $unscheduled->get_error_code() );
 	}
 
+	/**
+	 * If the option "cron" is empty _get_cron_array should return false!!
+	 *
+	 * @ticket 53651
+	 * @covers ::_get_cron_array
+	 */
+	public function test__get_cron_array(){
```

Comment:
```suggestion
	public function test__get_cron_array() {
```
(here and everywhere else)